maintainers/maintainer-list.nix

```diff
@@ -8659,6 +8659,11 @@
       fingerprint = "F5B2 BE1B 9AAD 98FE 2916  5597 3665 FFF7 9D38 7BAA";
     }];
   };
+  importantblimp = {
+    github = "importantblimp";
+    githubId = 20533356;
+    name = "GY";
+  };
```

Comment:
You do not need to add yourself in the maintainer list, if you do not add yourself in to the package `meta.maintainers` list

Reply:
Good to know, thanks - removed